Components/Image/image.html

```diff
@@ -0,0 +1,92 @@
+        <li class="menu-list"><a href="/index.html">Home</a></li>
+        <li class="menu-list"><a href="/Get-Started/introduction.html">Docs</a></li>
+        <li class="menu-list"><a href="https://github.com/som749898/sky-ui" target="_blank">Github</a></li>
+      </ul>
+    </header>
+    <div class="main-content flex">
+      <div class="component-list mg-1x">
+        <div class="get-started py-bottom-2x">
+          <h3 class="heading4">Get Started</h3>
+          <ul>
+            <li><a href="/Get-Started/introduction.html">Introduction</a></li>
+            <li><a href="/Get-Started/color.html">Color</a></li>
+            <li><a href="/Get-Started/typography.html">Typography</a></li>
+          </ul>
+        </div>
+        <div class="components">
+          <h3 class="heading4">Components</h3>
+          <ul>
+            <li><a href="/Components/Avatar/avatar.html">Avatar</a></li>
+            <li><a href="/Components/Alert/alert.html">Alert</a></li>
+            <li><a href="/Components/Badge/badge.html">Badge</a></li>
+            <li><a href="/Components/Button/button.html">Button</a></li>
+            <li><a href="#">Card</a></li>
+            <li><a href="/Components/Image/image.html">Image</a></li>
+            <li><a href="/Components/Input/input.html">Input</a></li>
+            <li><a href="#">List</a></li>
+            <li><a href="#">Toast</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="content mg-1x">
+        <h1 class="heading2 py-bottom-1x">Responsive Image</h1>
+        <p class="text-size py-bottom-3x">Responsive Image Components are used to dsiplay Images on websites, which are screen responsive. They expand upto the width of the container they are in.</p>
+        <h2 class="heading3 py-bottom-1x">Square Image</h2>
+        <p class="text-size py-bottom-1x">Square Images are the square shaped, with a default dimension of 150px X 150px.</p>
+        <div class="background-cover flex horizontal-center vertical-center gap-3x py-3x mg-bottom-1x">
+          <img class="img" src="/assets/img/image/image1.jfif" alt="image">
+          <img class="img" src="/assets/img/image/image2.jfif" alt="image">
+        </div>
+        <pre class="mg-bottom-3x"><code class="language-html">
+          &ltimg class="img" src="..." alt="image">
+          &ltimg class="img" src="..." alt="image">
+        </code></pre>
+        <h2 class="heading3 py-bottom-1x">Round Image</h2>
+        <p class="text-size py-bottom-1x">Round Images are the circular, with a default dimension of 150px X 150px.</p>
+        <div class="background-cover flex horizontal-center vertical-center gap-3x py-3x mg-bottom-1x">
+          <img class="img round" src="/assets/img/image/image3.jfif" alt="image">
+          <img class="img round" src="/assets/img/image/image4.jfif" alt="image">
+        </div>
+        <pre class="mg-bottom-3x"><code class="language-html">
+          <img class="img round" src="..." alt="image">
```

Comment:
This code section was not visible to me, please check once.

Reply:
I have corrected the code please review it again